api/metadata_processor_test.go

```diff
@@ -2125,6 +2144,18 @@ func (suite *MetadataProcessorSuite) someTags() []string {
 	return uids
 }
 
+func (suite *MetadataProcessorSuite) someLikutim() []string {
```

Comment:
better get input with list of sources and create likutim units for them.
This way no need for `createLikutim(exec boil.Executor)`

Reply:
changed